src/commands/community/track/sales.ts

```diff
@@ -27,15 +30,18 @@ const command: Command = {
       .fetch(channelId)
       .catch(() => undefined)
     if (!chan) throw new InvalidInputError({ message: msg })
+    const addr = args[3]
+    const platform = args[4]
+    const guildId = msg.guild.id
 
-    await Config.updateSalesConfig(msg.guildId, channelId)
+    await community.createSalesTracker(addr, platform, guildId, channelArg)
```

Comment:
handle error from api
- if 200 -> return success message
- other response -> return default error view of mochi